project/Dependencies.scala

```diff
@@ -13,7 +13,7 @@ object Dependencies {
 
   val fusekiImage =
     "daschswiss/apache-jena-fuseki:2.1.2" // should be the same version as in docker-compose.yml, also make sure to use the same version when deploying it (i.e. version in ops-deploy)!
-  val sipiImage = "daschswiss/sipi:3.8.1" // base image the knora-sipi image is created from
+  val sipiImage = "daschswiss/sipi:3.8.2" // base image the knora-sipi image is created from
```

Comment:
Should we also replace the `latest` tag  in the `docker-compose.yml`?

Reply:
I'm not sure which is worse. Bumping Sipi would then require changes in two places. If it gets forgotten, then the local development environment is out of sync.

Reply:
I guess it would then be the same as for Fuseki.

Reply:
@seakayone shell I change docker-compose.yml?

Reply:
@seakayone ah, it is not possible to change it. The version in docker-compose.yml needs to correspond to the knora-sipi version, which is built in the dsp-api repository. Basically, this would need to be changed manually on every commit.